### DISCUSSION TITLE
posts/odoo/Odoo-ir-sequence

### DISCUSSION THREAD

**Discussion by giscus** [dreamhunter2333/dreamhunter-blog#15] [General]
# posts/odoo/Odoo-ir-sequence

你指尖跃动的电光，是我此生不变的信仰

https://dreamhunter2333.com/posts/odoo/Odoo-ir-sequence.html

**Comment by dreamhunter2333** [comment]
> 迁移数据
> "createdAt": "2020-05-17T22:34:52.247Z",
>  "ip": "181.199.33.198",
> "nick": "Anonymous",
> "ua": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36",

aaaa
